docs/python/training/conf.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+
+# -- Path setup --------------------------------------------------------------
+
+import os
+import sys
+
+# ORTModule source-code folder
+sys.path.insert(0, os.path.abspath(os.path.join('..','..','..','..','orttraining','orttraining','python','training')))
+print(os.path.abspath(os.path.join('..','..','..','..','orttraining','orttraining','python','training')))
```

Comment:
Is this a debug print statement? Shall we remove it or print a more specific message?

Reply:
ooops, it is. I will remove in the next PR, in which I will integrate with Liqun work